### ISSUE TITLE
Support for responsive images is needed.

### ISSUE THREAD

**Issue by Graham-72** [backdrop-contrib/views_slideshow#3]
There is a patch for this available for the Drupal 7 version of views_slideshow_cycle and it would be good to add it to this Backdrop version.

**Comment by herbdool** [comment]
Just tried out 3.1.1 -- having a width: 100% on images forces the images to use the whole slideshow and beyond whatever natural size the image has. For example, I've got a slideshow that spans the whole width of the site and it also includes other fields which are pushed down by this.

**Comment by herbdool** [comment]
My suggested patch (includes removing width on thumbnails):

```
diff --git a/contrib/views_slideshow_cycle/views_slideshow_cycle.css b/contrib/views_slideshow_cycle/views_slideshow_cycle.css
index c7ce890..62feb5a 100644
--- a/contrib/views_slideshow_cycle/views_slideshow_cycle.css
+++ b/contrib/views_slideshow_cycle/views_slideshow_cycle.css
@@ -9,7 +9,6 @@
 /* Pager Settings */
 
 .views_slideshow_pager_field img {
-  width: 75px;
   height: 75px;
   margin-right: 5px;
   float: left;
@@ -46,7 +45,6 @@
 }
 .views_slideshow_cycle_main .field-content img {
   max-width: 100%;
-  width: 100%;
   height: auto;
   margin: 0;
   padding: 0;
```

**Comment by Graham-72** [comment]
@herbdool thanks for this and for pointing out the problem. However, at the moment I am not seeing this. 
One thing I notice is that in the Backdrop version we do not seem to be getting the class .field-content added to the image fields whereas it is in Drupal when Fields: Settings 'Provide default field wrapper elements' is ticked. This being the case the patch I applied would presumably not work anyway. Do you see the field-content class being applied to the image?

**Comment by herbdool** [comment]
I see .field-content wrapping the img.

Your previous commit applied the width: 100% to `.views_slideshow_cycle_main .field-content img` so I presume you were seeing it. At any rate, forcing an image to fill up the width of the slideshow means that it gets stretched beyond it's natural size and forces all text below even if that wasn't intended.

**Comment by Graham-72** [comment]
>I presume you were seeing it

It seems I was making a mistake here and I have added a fresh test of views slideshow to check things out. This confirms that the field-content class is not added if 'provide default field wrapper elements' is not ticked, and so the fix I have proposed would not work in all circumstances.

However, even without the default field wrapper elements, in Backdrop the slideshow image is still responsive because a stylesheet in core (base.css) is adding max-width: 100%; height: auto; to the img tag.

So it would seem that in Backdrop we do not require this commit that I have copied from Drupal. I had better remove it.
